tests/test_taskgroups.py

```diff
@@ -582,7 +621,11 @@ async def child(fail):
 
 
 async def test_cancel_cascade():
+    tg2 = None
```

Comment:
Move this to the top of the main code block (after the function definitions) for consistency.

Reply:
Done